projects/ontimize-web-ngx/src/lib/components/table/extensions/sort/o-mat-sort.ts

```diff
@@ -1,5 +1,5 @@
-import { Directive, EventEmitter, Output } from '@angular/core';
-import { MatSort, MatSortable } from '@angular/material';
+import { Directive, ElementRef, EventEmitter, Output, Renderer2 } from '@angular/core';
```

Comment:
Is correct the import of _Renderer2_? Is it being used anywhere?

Reply:
It was an unused import